Source/Scene/BoxEmitter.js

```diff
@@ -0,0 +1,45 @@
+/*global define*/
+define([
```

Comment:
I pushed docs for the emitter classes in 5b95fa232ec545adc88884fd069b864d172ce971 since they should be public.

The plumbing for the entity framework really did seem like lots of duplicate work, but I tried to keep it similar to the way the other visualizers worked.  Basically ParticleSystemGraphics is just storing properties for the ParticleSystem primitive.  The only interesting thing the visualizer does is ties the entity's model matrix to the particle system primitive's model matrix.  

Originally I just had the sandcastle demo using the ParticleSystem primitive directly and just tying it to the entity in a preFrame callback and the code was pretty straightforward, although not as obvious as the entity framework.

If you can think of a way to slim down the ParticleSystemGraphics I'd be all for it, I was just trying to keep it similar to the other things in the entity framework.